### PR TITLE
Update wallet adapter docs on AIP-62 standard

### PR DESCRIPTION
**Pull request by 0xmaayan**
### Checklist

- Do all Lints pass?
  - [] Have you ran `pnpm spellcheck`?
  - [] Have you ran `pnpm fmt`?
  - [] Have you ran `pnpm lint`?

### CODE REVIEW

**Review by gregnazario — APPROVED** [review]
can you update nextra as well?

**Comment by 0xmaayan** [comment]
> can you update nextra as well?

Done. also, added more docs on wallets-opt-in support - please review

File: apps/docusaurus/docs/integration/wallet-adapter-for-dapp.md [hunk 1]
```diff
@@ -142,3 +143,18 @@ const wallets = [new PetraWallet()];
...
 </AptosWalletAdapterProvider>;
 ```
+
+#### How to opt-in to show only specific wallets?
+
+Since the discovery of uninstalled AIP-62 wallets is challenging for a dapp, the adapter supports a discovery logic to include any AIP-62 wallet that is not installed on the user's machine and adds it by default to the final wallets array used by the dapp.
+
+Sometimes, a dapp might want to specifically opt in to certain wallets and display only those. For this, you should be able to pass an `optInWallets` array to the `AptosWalletAdapterProvider` with the names of the wallets you wish to include in your dapp.
+
+```js
```
Comment (gregnazario) [inline apps/docusaurus/docs/integration/wallet-adapter-for-dapp.md:153]: This should probably be `tsx` or `jsx`

**Review by gregnazario — APPROVED** [review]

File: apps/docusaurus/docs/integration/wallet-adapter-for-dapp.md (hunk 1 above)
Comment (0xmaayan) [reply apps/docusaurus/docs/integration/wallet-adapter-for-dapp.md]: done

**Pull request merged**